src/Authentication/Controllers/AuthenticationController.cs

```diff
@@ -708,6 +720,16 @@ private static bool HasServiceOwnerScope(ClaimsPrincipal originalPrincipal)
             return false;
         }
 
+        private static bool HasAltinnScope(string scope)
+        {
+            if (scope == null)
```

Comment:
redundant if when using nullable and `?? false` in line 720

Reply:
removed